README.md

```diff
@@ -22,7 +22,7 @@ For projects with a **feature based structure** this will be a problem, since yo
 |&nbsp;&nbsp;&nbsp;&nbsp;_foo.scss   | `url(bar.png)`         |
 |&nbsp;&nbsp;&nbsp;&nbsp;bar.png     |                        |
 
-Intuatively we want the assets in partial `_foo.scss` relative to the partial, meaning `url(bar.png)`.
```

Comment:
This text also appears in `packages/resolve-url-loader/README.md`. Could I please get you to repeat the fix there.